### DISCUSSION TITLE
How to completely disable pagination in a table?

### DISCUSSION THREAD

**Discussion by SpeedyInternet** [Power-Components/livewire-powergrid#1358] [Q&A] [answered]
Greetings, I am using PowerGrid but I have a conflict with the page, when I am on page 2 or 3 PowerGrid takes the value of the "page" variable from the url and affects the page page of the table.
How can I disable pagination? Or configure it so that it doesn't take the "page" variable from the url.

I appreciate your valuable help please.
![Screenshot_1](https://github.com/Power-Components/livewire-powergrid/assets/139509539/ea976706-6574-4615-8468-1cb620526752)

**Comment by dansysanalyst** [comment, marked as answer]
Hi @SpeedyInternet,

We are discussing this issue internally, if it is possible to improve PowerGrid without generating a breaking change.

Currently, it is not possible to change the `pageName` in a PowerGrid component.

So, this is not a direct answer to your question, but perhaps you can have a custom `pageName` for your main component to avoid the collisions.  For example, it can be `productPage`.

You can see more at Livewire documentation on [multiple paginators](https://livewire.laravel.com/docs/pagination#multiple-paginators).

Another possibility is to disable URL pagination in your component using Livewire's `WithoutUrlPagination` Trait.

```php
// app/Livewire/MyPowergridTable.php

class MyPowergridTable extends PowerGridComponent
{
    use \Livewire\WithoutUrlPagination;
    
     //...
```

**Comment by dansysanalyst** [comment]
@SpeedyInternet this feature is on its way to be merged and released.

Thanks to @luanfreitasdev  (PR https://github.com/Power-Components/livewire-powergrid/pull/1364)
